src/main/java/com/example/bcsd/controller/ArticleController.java

```diff
@@ -0,0 +1,68 @@
+package com.example.bcsd.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.bcsd.dto.RequestDto.ArticleCreateRequestDto;
+import com.example.bcsd.dto.RequestDto.ArticleUpdateRequestDto;
+import com.example.bcsd.dto.ResponseDto.ArticleResponseDto;
+import com.example.bcsd.service.ArticleService;
+
+@RestController
+@RequestMapping("/articles")
+public class ArticleController {
+
+    @Autowired
+    private ArticleService articleService;
+
+    @GetMapping
+    public ResponseEntity<List<ArticleResponseDto>> getArticles() {
+        List<ArticleResponseDto> responseDtos = articleService.getArticles();
```

Comment:
개인적인 느낌이지만, DTO를 명시하지 않는게 훨씬 깔끔할 거 같아요. 이미 사용하는 시점에서 DTO를 이용한다고 생각하기 때문..!
```suggestion
        List<ArticleResponse> responses = articleService.getArticles();
```